README.md

```diff
@@ -1,37 +1,41 @@
-# skeleton
+# @fastify/deepmerge
 
-Template repository to create standardized Fastify plugins.
+![CI](https://github.com/fastify/deepmerge/workflows/CI/badge.svg)
+[![NPM version](https://img.shields.io/npm/v/@fastify/deepmerge.svg?style=flat)](https://www.npmjs.com/package/@fastify/deepmerge)
+[![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](https://standardjs.com/)
 
-# Getting started
+Merges the enumerable properties of two or more objects deeply.
 
-- Click on `Use this template` above to create a new repository based on this repository.
+### Install
+```
+npm i @fastify/deepmerge
+```
+
+### Usage
```

Comment:
Could we add the bench results?
Why users should use this module instead of others?

Reply:
Done. :)